db/structure.sql

```diff
@@ -351,6 +351,19 @@ CREATE TABLE public.communications (
 );
 
 
+--
+-- Name: credentials; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.credentials (
+    name character varying NOT NULL,
```

Comment:
I don't see any artifact in the `structure.sql` that indicates that `name` is a primary key, or that it's uniqueness is enforced.

Reply:
PS why make the name the unique id? Any reason to deviate from the default?

Reply:
I'm thinking of this table like a key value store, given we'll primary access data using the name. Could leave a primary uuid key in and not use it to access data I guess.

Reply:
Yeah that seems more "boring", less likely to have any gotchas. We can always introduce a `fetch` helper that'll give us a convenient interface to fetch these values.